server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -192,6 +193,41 @@ public CompletableFuture<Token> updateToken(ServiceRequestContext ctx,
         );
     }
 
+    /**
+     * PATCH /tokens/{appId}/permission
```

Comment:
Should we use permission or level?

Reply:
I will change "permission" to "level" in the comment.